src/main/java/org/jenkinsci/plugins/workflow/log/OutputStreamTaskListener.java

```diff
@@ -73,7 +75,9 @@ public interface OutputStreamTaskListener extends TaskListener {
                 listener.getClass() == StreamTaskListener.class ||
                 listener.getClass() == LogTaskListener.class ||
                 listener.getClass() == StreamBuildListener.class ||
-                listener.getClass() == BuildListenerAdapter.class;
+                listener.getClass() == BuildListenerAdapter.class ||
+                listener.getClass() == TeeTaskListener.class;
```

Comment:
This does not make any sense as written. The point of this list is to enumerate classes which are defined in Jenkins core and thus cannot be fixed to be implemented in a way that uses an interface defined in this plugin. That obviously does not apply to `TeeTaskListener` which is implemented in this plugin.